### PR TITLE
Fix preposition for exclusive fullscreen prompt

### PR DESCRIPTION
**Pull request by CenTdemeern1**
I mistakenly PRed this to osu-resources originally (https://github.com/ppy/osu-resources/pull/225)

### CODE REVIEW

**Review by bdach — DISMISSED** [review]

**Comment by loma-baldsson** [comment]
Try not to submit a pull request from master (the reasoning can be found [here](https://github.com/ppy/osu/blob/master/CONTRIBUTING.md#i-would-like-to-submit-a-pull-request))

**Comment by CenTdemeern1** [comment]
> Try not to submit a pull request from master (the reasoning can be found [here](https://github.com/ppy/osu/blob/master/CONTRIBUTING.md#i-would-like-to-submit-a-pull-request))

Ah right, topic branches.
Well, I don't think I can or should change it now, but I'll keep that in mind for next time.

**Review by Joehuu — DISMISSED** [review]

**Review by peppy — APPROVED** [review]
I guess

**Comment by peppy** [comment]
Well, if we're changing this shouldn't the trnaslation directly below it also be fixed?

**Review by peppy — CHANGES_REQUESTED** [review]
As proposed

**Comment by CenTdemeern1** [comment]
I also removed the word "still" from the second translation because I don't think it made sense for it to be there at all

File: osu.Game/Localisation/LayoutSettingsStrings.cs [hunk 1]
```diff
@@ -20,9 +20,9 @@ public static class LayoutSettingsStrings
         public static LocalisableString OsuIsRunningExclusiveFullscreen => new TranslatableString(getKey(@"osu_is_running_exclusive_fullscreen"), @"osu! is running in exclusive fullscreen, guaranteeing low latency!");
 
         /// <summary>
-        /// "Unable to run exclusive fullscreen. You&#39;ll still experience some input latency."
+        /// "Unable to run in exclusive fullscreen. You&#39;ll experience some input latency."
         /// </summary>
-        public static LocalisableString UnableToRunExclusiveFullscreen => new TranslatableString(getKey(@"unable_to_run_exclusive_fullscreen"), @"Unable to run exclusive fullscreen. You'll still experience some input latency.");
+        public static LocalisableString UnableToRunExclusiveFullscreen => new TranslatableString(getKey(@"unable_to_run_exclusive_fullscreen"), @"Unable to run in exclusive fullscreen. You'll experience some input latency.");
```
Comment (peppy) [inline osu.Game/Localisation/LayoutSettingsStrings.cs:25]: How about "You may experience some input latency"?

Comment (CenTdemeern1) [reply osu.Game/Localisation/LayoutSettingsStrings.cs:25]: Oh yeah, that makes sense

**Review by peppy — APPROVED** [review]

**Pull request merged**